tree/ntuple/v7/inc/ROOT/RColumn.hxx

```diff
@@ -39,7 +39,7 @@ namespace ROOT::Experimental::Internal {
 // clang-format on
 class RColumn {
 private:
-   ENTupleColumnType fType;
+   ROOT::ENTupleColumnType fType;
```

Comment:
Question here and everywhere else: Do we want to qualify the types in our own classes? I think it should not be needed due to `namespace` nesting, and it becomes redundant when we move other classes. Will we remove them later on?

Reply:
As discussed in the IO meeting, it's needed because otherwise it will pick up the (deprecated) `using` declarations in `ROOT::Experimental`. The approach may be to remove the deprecation message for now and only add it back once all classes are moved until we remove the `using` declarations before branching.